app.cr

```diff
@@ -9,13 +9,20 @@ def headers
   headers
 end
 
-def crystal_repos
+def crystal_repos(sort)
   client = HTTP::Client.new("api.github.com", 443, true)
-  response = client.get("/search/repositories?q=language:crystal&per_page=100", headers)
+  response = client.get("/search/repositories?q=language:crystal&per_page=100&sort=#{sort}", headers)
```

Comment:
That was my first thought, but we can think about it on another pull request. I perfectly fine with that.